Loenn/lang/en_gb.lang

```diff
@@ -0,0 +1,133 @@
+mods.FactoryHelper.name=FactoryHelper
```

Comment:
This is the display name, so would change to "Factory Helper"

Reply:
shows up as FactoryHelper in Ahorn. i have considered it though, should this be a general guideline for display names?

Reply:
I think that would make sense, yeah